ceph_deploy/hosts/fedora/__init__.py

```diff
@@ -18,7 +18,7 @@ def choose_init():
 
     Returns the name of a init system (upstart, sysvinit ...).
     """
-    return 'sysvinit'
+    return 'systemd'
```

Comment:
Just to follow up - I was asking if we could add that conditional to this PR.

But, I'm happy to merge this in and then add the conditional on top.

I think the likely way forward for this is that this set of changes will go into a 1.5.27 version, and then we can focus on sweeping systemd changes for 1.5.28.